nodes/particles/particle_info.py

```diff
@@ -44,6 +49,17 @@ def getExecutionCode(self):
         if isLinked["previousRotation"]: lines.append("    previousRotation = mathutils.Vector(particle.prev_rotation.to_euler())")
         if isLinked["previousVelocity"]: lines.append("    previousVelocity = particle.prev_velocity")
         if isLinked["previousAngularVelocity"]: lines.append("    previousAngularVelocity = particle.prev_angular_velocity")
+        
+        hair = ["hairTime", "hairWeight", "hairPoints", "hairPointsLocal"]
+        if any ([isLinked[item] for item in hair]):
```

Comment:
I don't use spaces between function names and its parameters "any{nospace}(...)"